csharp/Platform.IO/FileHelpers.cs

```diff
@@ -86,12 +242,48 @@ public static void SetSize(string path, long size)
             }
         }
 
+        /// <summary>
+        /// <para>Removes all files from the <paramref name="directory"/>.</para>
+        /// <para>Удаляет все файлы из директории <paramref name="directory"/>.</para>
```

Comment:
`Удаляет все файлы из директории находящейся по пути <paramref name="directory"/>.`

We should add the word path here, because the argument name does not contain it.